scripts/parse_revng_model.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+import sys
+import yaml
+
+from revng import model
+
+m = yaml.load(sys.stdin, Loader=model.YamlLoader)
+
+print(yaml.dump(m, Dumper=model.YamlDumper))
```

Comment:
* `def main`
* What's the purpose of this tool? Is this is for testing only? Add docs or consider inlining it in `scripts/run-serialization-roundtrip-test.sh` with `python -c`.

Reply:
Just for testing, I'll inline it